packages/paste-website/src/components/assistant/AssistantComposer.tsx

```diff
@@ -30,23 +30,25 @@ export const AssistantComposer: React.FC<{ onMessageCreation: (message: string,
   };
 
   return (
-    <ChatComposer
-      maxHeight="size10"
-      config={{
-        namespace: "foo",
-        onError: (error: Error) => {
-          throw error;
-        },
-      }}
-      ariaLabel="Message"
-      placeholder="Type here..."
-      onChange={handleComposerChange}
-      ref={editorRef}
-    >
-      <ClearEditorPlugin />
-      <SendButtonPlugin onClick={submitMessage} disabled={selectedThread == null} />
-      <EnterKeySubmitPlugin onKeyDown={submitMessage} />
-      <FocusComposerPlugin selectedThread={selectedThread} />
-    </ChatComposer>
+    <ChatComposerContainer variant="contained">
+      <ChatComposer
+        maxHeight="size10"
+        config={{
+          namespace: "foo",
+          onError: (error: Error) => {
+            throw error;
+          },
+        }}
+        ariaLabel="Message"
+        placeholder="Type here..."
+        onChange={handleComposerChange}
+        ref={editorRef}
+      >
+        <ClearEditorPlugin />
+        <SendButtonPlugin onClick={submitMessage} disabled={selectedThread == null} />
```

Comment:
suggestion: Can you use the action group and button like this example in our docs: https://paste.twilio.design/components/chat-composer#chat-composer-contained-variant

The current way has the button inside the text areas and will block text.